sumologic/resource_sumologic_monitors_library_folder.go

```diff
@@ -92,18 +92,24 @@ func resourceSumologicMonitorsLibraryFolder() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+
 			"post_request_map": {
 				Type:     schema.TypeMap,
 				Optional: true,
 				Elem: &schema.Schema{
 					Type: schema.TypeString,
 				},
 			},
+
+			"obj_permission": GetCmfFgpObjPermSetSchema(),
```

Comment:
nit: Just call it "permission"?

Reply:
OK. I think I would do the renaming, after first round of Git Hub Integration Tests got executed successfully.

Reply:
On second thought, I would stay with "obj_permission". 
Consider this TF example: 
```
resource "sumologic_monitor_folder" "test_monitorfolder" {
		name        = "terraform_test_monitorfolder_%s"
		description = "terraform_test_monitorfolder_desc"
		obj_permission {
		  subject_type = "role"
		  subject_id = sumologic_role.tf_test_role_01.id 
		  permissions = ["Create","Read","Update","Delete"] 
		}
		obj_permission {
		  subject_type = "role"
		  subject_id = sumologic_role.tf_test_role_02.id
		  permissions = ["Create", "Read"]
		}
}
```

If I rename the outer "obj_permission" to "permission", the above example would look like this:  
```
resource "sumologic_monitor_folder" "test_monitorfolder" {
		name        = "terraform_test_monitorfolder_%s"
		description = "terraform_test_monitorfolder_desc"
		permission {
		  subject_type = "role"
		  subject_id = sumologic_role.tf_test_role_01.id 
		  permissions = ["Create","Read","Update","Delete"] 
		}
		permission {
		  subject_type = "role"
		  subject_id = sumologic_role.tf_test_role_02.id
		  permissions = ["Create", "Read"]
		}
}
```
The outer "permission" will collide the inner "permissions" array. It may be confusing during customer support communication.

Reply:
ok, let's keep it.